File: atbash.c
#include <stdio.h>
#include <ctype.h>

char atbash(char c) {
    if (isalpha(c)) {
        if (islower(c)) {
            return 'a' + 'z' - c;
        } else {
            return 'A' + 'Z' - c;
        }
    }
    return c;
}

int main() {
    int c;
    while ((c = getchar()) != EOF) {
        putchar(atbash(c));
    }
    return 0;
}

File: atbash.py
import sys

def atbash(c):
    if c.isalpha():
        if c.islower():
            return chr(ord('a') + ord('z') - ord(c))
        else:
            return chr(ord('A') + ord('Z') - ord(c))
    return c

def main():
    while True:
        c = sys.stdin.read(1)
        if not c:
            break
        sys.stdout.write(atbash(c))

if __name__ == '__main__':
    main()
